tests/unit/selectmenu/selectmenu_core.js

```diff
@@ -2,6 +2,22 @@
 
 module( "selectmenu: core" );
 
+test( "markup structure", function() {
+	expect( 4 );
+
+	var element = $( "#files" ).selectmenu(),
+		button = element.selectmenu( "widget" ),
+		menu = element.selectmenu( "menuWidget" ),
+		menuWrap = menu.parent();
+
+	ok( button.is( ".ui-selectmenu-button.ui-selectmenu-button-closed.ui-widget.ui-state-default" ),
```

Comment:
This either needs to be converted to individual `hasClass` calls as well (with the same caveat as above) or use `equal` with `.attr( "class" )`.